doc/README.md

```diff
@@ -78,6 +79,7 @@ response_type=code
 &state=0facda3319
 &make=TESLA
 &single_select=true
+&flags=country:DE
```

Comment:
Can you a couple more feature flag examples to "flags" parameter. Something like '&flags=country:DE%20test1:value1%20test2:value2'. Developers could understand that '%20" is html encoded value for space but we could put a note saying so. What are your thoughts about this?

Reply:
I will add another example! I will keep it un-encoded though like the example for the `scope` parameter a couple of lines above it.